src/components/ProvinceDropdown/ProvinceDropdown.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import Select from "material-ui/Select"
+import PropTypes from "prop-types"
+import { MenuItem } from "material-ui/Menu"
+import { InputLabel } from "material-ui/Input"
+import { FormControl, FormHelperText } from "material-ui/Form"
+import ProvinceDropdownData from "./province.data"
+
+export default class ProvinceDropdown extends React.Component {
+  state = {
+    province: "",
+  }
+
+  handleProvinceChange = event => {
+    this.setState({ province: event.target.value }, () => {
+      this.props.onchange(this.state.province)
+    })
+  }
+
+  render() {
+    const formControl = {
```

Comment:
Again, using style objects in render will cause it to be reconstructed on every re-render. You should move the formControl and select style object out of render, and preferably on top of the file.